yorkie/rpc/server.go

```diff
@@ -125,3 +136,26 @@ func (s *Server) listenAndServeGRPC() error {
 
 	return nil
 }
+
+// Validate validates port number and checks if files exists
+func (c *Config) Validate() error {
+	var err error
+	if c.Port < 0 || 65535 < c.Port {
+		return fmt.Errorf("%w: %d", ErrWrongPort, c.Port)
+	}
+
+	// when specific cert or key file are configured
+	if c.CertFile != "" {
+		if _, err = os.Stat(c.CertFile); err != nil {
+			return fmt.Errorf("%w: %s", ErrNoCertFile, c.CertFile)
+		}
```

Comment:
`os.Stat` can return various errors, and whether the file exists can be checked with `os.IsNotExist`. It would be better to use sentinel error in a broader sense here.

```suggestion
		if _, err = os.Stat(c.CertFile); err != nil {
			return fmt.Errorf("%w: %s", ErrInvalidCertFile, c.CertFile)
		}		
```

Reply:
I didn't know that! Thank you.